### ISSUE TITLE
Convert alibi detect model (tf backend)

### ISSUE THREAD

**Issue by Sofien1945** [SeldonIO/alibi-detect#483]
I have trained a VAR model and i would like to convert it to onnx to deploy on an edge device. Is it possible to use tfonnx conversion for this purpose and keep using alibi detect methods such predict, plot_feature_outlier_tabular

**Comment by jklaise** [comment]
Hi @Sofien1945, do you mean a `VAE` outlier detector model? Could this link be helpful: https://github.com/SeldonIO/alibi-detect/issues/279#issuecomment-870470095

We haven't tested conversion to `onnx` for alibi detect models explicitly but I don't think it would work because our models are basically custom Python objects that may contain `tensorflow` models inside them.

**Comment by Sofien1945** [comment]
Yes it s VAR outlier to detect anomaly inside image. I am trying to convert it using tensorflow onnx converter and i willcshare my result thank you
